OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -334,6 +334,9 @@ public Sdl2PlatformWindow(Size requestEffectiveWindowSize, WindowMode windowMode
 				Console.WriteLine($"Using window scale {windowScale:F2}");
 			}
 
+			if (Platform.CurrentPlatform == PlatformType.OSX)
```

Comment:
There's already a macOS specific branch above (line 267) - any reason why this can't be included there?